Makefile

```diff
@@ -35,7 +35,7 @@ DIST_FILE=aci-containers.tgz
 
 DOCKER_HUB_ID ?= noiro
 DOCKER_TAG ?=
-BUILD_CMD ?= go build -v -trimpath
+BUILD_CMD ?= go build -v -trimpath -race
```

Comment:
we can add a new build command option here, but the existing one should not be modified...